static/src/components/__tests__/AnnotationButton.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { mount } from "@cypress/vue";
+import AnnotationButton from "../AnnotationButton.vue";
+
+describe("Button", () => {
+  it("exists", () => {
+    mount(AnnotationButton), { probs: { text: String } };
+  });
+
+  it("renders properly", () => {
+    mount(AnnotationButton), { probs: { text: String } };
+    cy.get("button").should("be.visible");
+  });
+
+  it("is not disabled", () => {
+    mount(AnnotationButton), { probs: { text: String } };
+    cy.get("button").should("not.be.disabled");
+  });
+
+  it("shows specific label", () => {
+    const str = "Label: ";
```

Comment:
Maybe it would be useful to use a different `str` here because "Label: " is included in the button by default. So then we could test that `str` is really displayed by the button